pkg/transformers/builder/transformer_builder.go

```diff
@@ -44,6 +44,8 @@ func New(cfg *transformers.Config) (transformers.Transformer, error) {
 		return transformers.NewPhoneNumberTransformer(cfg.Parameters)
 	case transformers.Masking:
 		return transformers.NewMaskingTransformer(cfg.Parameters)
+	case transformers.Noop:
```

Comment:
What if we didn't need a noop implementation at all? Could we keep the "noop" transformer as a configuration concept, that doesn't translate into the transformer implementation but it's caught early by the validation?

Maybe the validation step can receive the config as well as the transformer map and can validate both? Or we could potentially unify the parsing of the rules into the transformers map and the validation in one step?